content/general/functions.textile

```diff
@@ -12,73 +12,61 @@ Generally, customers who want to receive events as they happen, use our "message
+* "MessagePack":https://msgpack.org: An efficient binary serialization format that is similar to JSON, but faster and smaller.
 
 h2. Envelopes
 
-Messages sent by Reactor Functions can be contained within an envelope. This can be useful for clients which are unable to access response headers or status codes.
-
+Messages sent by Reactor Functions can be contained within an envelope. This can be useful for clients which are unable to access response headers or status codes, or wish to receive more metadata about the payload.
 
 h2. Payload types
 
 Ably currently supports two types of data that can be delivered via our Functions:
-
 * "Messages":#messages - messages trigger function calls as soon as they are published on a channel
 * "Presence events":#presence - when clients enter, update their data, or leave channels, the presence event triggers a function call
 
-You can "configure Functions":#configure from the Reactor tab in your "account dashboard":https://support.ably.io/solution/articles/3000048664-how-do-i-access-my-account-dashboard on a per app basis which can apply to one or more channels in that app. Reactor Functions can be filtered by channel naming using a regular expression.
-
-
+You can "configure Functions":#configure from the "Reactor tab in your app":https://support.ably.io/solution/articles/3000074406-how-to-reach-the-reactor-page on a per app basis which can apply to one or more channels in that app. Reactor Functions can be filtered by channel naming using a regular expression, for example @^click_.*_mouse$@.
 
 h2(#messages). Messages
 
 If you are unfamiliar with how channels work in Ably, we recommend you first review the "realtime channel documentation":/realtime/channels-messages#channels.
 
 When a message is published on a channel, a Reactor Function message event is triggered, which in turn will send a message to your server-less function.
 
-Every request will contain your message within the @body@ part of the request. The contents of the @body@ depends on the format of your messages, and whether you've used an envelope or not.
-
-* No Envelope - Without an envelope, the contents of @body@ will simply be the contents message that was sent on the Ably Channel. 
-
-* Envelope - If you use an envelope, the @body@ will contain a bunch of extra attributes, which are the following:
-
+Every request will contain your message within the "HTTP message body":https://en.wikipedia.org/wiki/HTTP_message_body part of the request. The contents of the @HTTP message body@ depends on whether you've used an envelope or not. If you don't use an envelope, the @HTTP message body@ will only contain the @data@ payload from your message.
 
+@Enveloped@ - when the Reactor rule is configured to use an enveloper, the @HTTP message body@ will contain a bunch of extra attributes, which are the following:
 - source := the source of the request, namely "@channel.message"
```

Comment:
Best to separate textile types with clear line breaks to avoid ambiguity about what may happen i.e. the line above is rendered as a `<p>`, yet this definition list may or may not render separately, so I find it's better to just make it clear where a group of marked up element start and end with white space.